pkg/kn/commands/service/service.go

```diff
@@ -15,7 +15,14 @@
 package service
 
 import (
+	"io"
+	"time"
+
 	"github.com/knative/client/pkg/kn/commands"
+	serving_kn_v1alpha1 "github.com/knative/client/pkg/serving/v1alpha1"
+
+	"fmt"
```

Comment:
i thought we were grouping std imports together?

Reply:
We are running now "goimports" unconditionally on every build. So we take exactly what is spit out there, that there is no ambiguity anymore. Is there anything we could optimize ?